src/Tribe/Tickets.php

```diff
@@ -176,6 +176,83 @@ public function get_ticket( $event_id, $ticket_id ) {
 		}
 
 		/**
+		 * Returns the html for the delete ticket link
+		 */
+		public function get_ticket_delete_link( $ticket = null ) {
+			if ( empty( $ticket ) ) {
+				return;
+			}
+
+			if ( apply_filters( 'tribe_tickets_current_user_can_delete_ticket', true, $ticket->ID, $ticket->provider_class ) ) {
+				$delete_link = sprintf( '<span><a href="#" attr-provider="%1$s" attr-ticket-id="%2$s" id="ticket_delete_%2$s" class="ticket_delete">' . esc_html__( 'Delete Ticket', 'event-tickets' ) . '</a></span>', $ticket->provider_class, $ticket->ID );
```

Comment:
for readability would suggest breaking this into multiple lines. you should be able to do that by making the `esc_html__( 'Delete Ticket', 'event-tickets' )` an argument for `%3$s`

```
<span><a href="#" attr-provider="%1$s" attr-ticket-id="%2$s" id="ticket_delete_%2$s" class="ticket_delete">%3$s</a></span>
```